### ISSUE TITLE
error while loading pvrecorder library in windows 10

### ISSUE THREAD

**Issue by goutham2688** [GiviMAD/rustpotter-cli#1]
Hey, 
thanks for this creating this project 👍  
the cli tools seems to be panicking due to lack of a dependent library being missing. (im using Windows 10)

```
.\rustpotter-cli_win_x86_64.exe devices
thread 'main' panicked at 'Failed to initialize recorder: RecorderError { status: LibraryLoadError, message: "Failed to load pvrecorder dynamic library: LoadLibraryExW failed" }', src\cli\devices.rs:19:6
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by goutham2688** [comment]
the error seems to be only in the exe i got from releases page, but when i build with cargo.. i get no errors.

**Comment by goutham2688** [comment]
ok, issue, is not seen only in debug version. but error is seen when built with "cargo bbuild -r"

**Comment by GiviMAD** [comment]
Thank you for reporting it.
Didn't have time to test it on windows, I was waiting for a friend feedback but I will give it a try.
This issue was fixed in the 0.4.0 release, by including the appropriate pv_recorder library in to the binary which get extracted to a temp dir and loaded. At least it's fixed for macOS and Linux.
Which version are you testing?

**Comment by GiviMAD** [comment]
I just confirm that it's failing in latest version.
Seem to be related on how windows handles the temp files. 
Looking at it.

| ok, issue, is not seen only in debug version. but error is seen when built with "cargo bbuild -r"

Yes in debug mode it's not necessary to handle the library load (handled by pv_recorder itself) so that stuff is disabled.

**Comment by goutham2688** [comment]
> Which version are you testing?

I'm using rustpotter-cli version 0.7.0

**Comment by GiviMAD** [comment]
@goutham2688, fixed on 0.8.0. Thank you again for reporting it.
